City-of-Helsinki/drupal-infofinland
# Feature/IFU-543: municipality form functionality

**How to test**
```
Make build
Make shell
Drush cim -y
Drush cr
```

Create basic page (kunta sivu) and choose kunta for it.
Add Kuntanosto paragraph to basic page (valtakunnallinen sivu).
Add the kuntasivu you just create to the Kuntanosto paragraph.
Go back to kuntasivu. Check if Nosto valtakunnalliselle sivulle appears to sidebar.
